routes/annotation.js

```diff
@@ -93,7 +93,7 @@ function post(req, res, next) {
                     arr.push(obj.key);
                     return arr;
                 }, []).join(',');
-                environment.annotations = body;
+                environment.annotations = body.sort(orderBy);
```

Comment:
doesn't look like these are being sorted correctly. Might need to add a test for them. 
http://mss-shipit-api.dev.services.ec2.dmtio.net/v1/shipment/deployit-trigger-tests/environment/alb-http-internal